src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,199 @@ Master Pre-Releases
+  `PR #9300 <https://github.com/pantsbuild/pants/pull/9300>`_
+
+* Remove deprecated `pants.backend.python` files moved to `pants.python` (#9238)
+  `PR #9238 <https://github.com/pantsbuild/pants/pull/9238>`_
+
+* Remove deprecated `--target-spec-file` and `--positional-arg-file` in favor of `--spec-file` (#9237)
+  `PR #9237 <https://github.com/pantsbuild/pants/pull/9237>`_
+
+* Remove deprecated `--owner-of` in favor of file arguments (#9235)
+  `PR #9235 <https://github.com/pantsbuild/pants/pull/9235>`_
+
+* Remove deprecated `--dependencies-internal-only` and `--dependencies-external-only` (#9258)
+  `PR #9258 <https://github.com/pantsbuild/pants/pull/9258>`_
+
+* Upgrade to pex 2.1.6. (#9255)
+  `PR #9255 <https://github.com/pantsbuild/pants/pull/9255>`_
+
+* Remove deprecated `--fmt-skip` and `--lint-skip` (#9254)
+  `PR #9254 <https://github.com/pantsbuild/pants/pull/9254>`_
+
+* Revert removing `--test-fast` and delay the removal from 1.27.0.dev0 to 1.28.0.dev0 (#9251)
+  `PR #9251 <https://github.com/pantsbuild/pants/pull/9251>`_
+
+* Revert "Load the bootstrapped zinc compiler from the zinc server's classpath. (#8753)" (#9226)
+  `PR #9226 <https://github.com/pantsbuild/pants/pull/9226>`_
+
+* Stop defaulting to `--transitive` and remove the option when it no-ops (#9236)
+  `PR #9236 <https://github.com/pantsbuild/pants/pull/9236>`_
+
+* Remove `--glob-expansion-failure` and deprecate `--files-not-found-behavior=ignore` (#9242)
+  `PR #9242 <https://github.com/pantsbuild/pants/pull/9242>`_
+
+* Remove `--test-fast` to always have better test caching (#9241)
+  `PR #9241 <https://github.com/pantsbuild/pants/pull/9241>`_
+
+* Remove deprecated `dict_option`, `list_option`, and `target_list_option` (#9243)
+  `PR #9243 <https://github.com/pantsbuild/pants/pull/9243>`_
+
+* Deprecate `--build-file-imports` and remove its deprecated value `allow` (#9234)
+  `PR #9234 <https://github.com/pantsbuild/pants/pull/9234>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Add support for `--dependencies-type=3rdparty` to `dependencies2` (#9225)
+  `PR #9225 <https://github.com/pantsbuild/pants/pull/9225>`_
+
+* pants build committing publish data for push of org.pantsbuild#zinc-compiler_2.12;0.0.20["ci skip"]
+
+* Support build file aliases in v2. (#9279)
+  `PR #9279 <https://github.com/pantsbuild/pants/pull/9279>`_
+
+* first implement v1 of robot impl (#8793)
+  `PR #8793 <https://github.com/pantsbuild/pants/pull/8793>`_
+
+* Prevent excessive uncached readlink's when using a symlinked workdir (#9270)
+  `PR #9270 <https://github.com/pantsbuild/pants/pull/9270>`_
+
+* [jvm] Add jvm_options to platform; use from junit and run (#9150)
+  `PR #9150 <https://github.com/pantsbuild/pants/pull/9150>`_
+
+* Enable zinc to log diagnostics for jvm languages (#9228)
+  `PR #9228 <https://github.com/pantsbuild/pants/pull/9228>`_
+
+Bugfixes
+~~~~~~~~
+
+* Make the platform a remote execution happened on available to `@rule`s. (#9198)
```

Comment:
Refactoring

Reply:
Bump on moving this to refactoring, ideally near the top because this is a new change compared to a planned removal.